string append_this
bool new_file
---
string appended_file

